addons/medical/functions/fnc_treatmentSAMSplintTimeout.sqf

```diff
@@ -0,0 +1,32 @@
+/*
+ * Author: Orbis2358
+ * Splint timeout check
+ *
+ * Arguments:
+ * 0: The patient <OBJECT>
+ * 1: SelectionName <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [patient, "SelectionName"] call ace_medical_fnc_treatmentSAMSplintTimeout
+ *
+ * Public: Yes
+ */
+
+#include "script_component.hpp"
+
+params ["_target", "_part", "_timeOld"];
+
+private _sam = _target getVariable [QGVAR(samSplint), [[0, 0], [0, 0], [0, 0], [0, 0], [0, 0], [0, 0]]];
+
+if (_sam select _part select 0 isEqualTo _timeOld) then {
```

Comment:
what does `sam` stand for? Looks like it should be called `splint`

Reply:
That was for SAM Splint, but fixed anyway